source/client/benchmark_client_impl.h

```diff
@@ -74,7 +74,7 @@ class Http1PoolImpl : public Envoy::Http::Http1::ProdConnPoolImpl {
 // --max-requests-per-connection may help as well, as doing periodically initiating new connections
 // may help the benchmark target by giving it an opportunity to rebalance.
 class Http2PoolImpl : public Envoy::Http::ConnectionPool::Instance,
-                      public Envoy::Http::ConnPoolImplBase {
+                      public Envoy::Http::Legacy::ConnPoolImplBase {
```

Comment:
Do we have a bug to remove legacy?

Reply:
Not yet -- but let me add one. It looks like some exciting stuff landed in Envoy :-)